<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sf::MiscellaneousPropertyPage</class>
 <widget class="sf::PropertyPage" name="sf::MiscellaneousPropertyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiscellaneousPropertyPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Object</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblObjectName">
        <property name="text">
         <string>&amp;Name</string>
        </property>
        <property name="buddy">
         <cstring>leObjectName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leObjectName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbAlignment">
     <property name="title">
      <string>Alignment</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="lblAlignmentHorizontal">
        <property name="text">
         <string>Horizontal</string>
        </property>
        <property name="buddy">
         <cstring>cbAlignmentHorizontal</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbAlignmentHorizontal"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblAlignmentVertical">
        <property name="text">
         <string>Vertical</string>
        </property>
        <property name="buddy">
         <cstring>cbAlignmentVertical</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbAlignmentVertical"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbLabel">
     <property name="title">
      <string>Label</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblBuddy">
        <property name="text">
         <string>&amp;Buddy</string>
        </property>
        <property name="buddy">
         <cstring>cbBuddy</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbBuddy"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblLabelText">
        <property name="text">
         <string>&amp;Text</string>
        </property>
        <property name="buddy">
         <cstring>leLabelText</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leLabelText"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>sf::PropertyPage</class>
   <extends>QWidget</extends>
   <header>misc/qt/PropertyPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
